nomenclature/definitions/region/subcountries.yaml

```
@@ -302,28 +302,58 @@ United Kingdom|North Sea:
     ehighway2050: 106_NS+107_NS+108_NS+109_NS
     plan4res: NSUK
     
-# List of electricity price regions in Norway
-Norway|Oslo:
-   definition: this is the electricity price region 1 in Norway 
+# List of subregions in Norway, designed to best decribe the hydropower system in Norway
+Norway|OSTLAND:
+   definition: this is region N1 in the HydroCen dataset, these regions are based on the norwegian electricicty regions but with finer granularity (and some changes) 
```

Not necessary to write "and some changes" to the definition, because future users won't know that there was an earlier version.

Please follow the convention used for the other sub-country regions, see the following suggestion
```suggestion
   definition: <please give a self-explaining definition, using NUTS-x codes if possible
   note: This region was defined in the HydroCen dataset based on the Norwegian electricity regions
   hydrocen: N1
```

---

Please implement this change for all regions, not just the one region where I proposed it

---

ok should be done